src/modules/app/containers/LocationLayout.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
```

Comment:
LocationLayout is part of the location route. It is ony used there.
I know this is problematic because of RouteFragments but we should refactor this then probably.

Reply:
no it's also used for events, disclaimer and search

Reply:
but again maybe the name is not quite clear